src/Discord/Internal/Types/Gateway.hs

```diff
@@ -51,7 +54,7 @@ data GatewayIntent = GatewayIntent
   , gatewayIntentWebhooks :: Bool
   , gatewayIntentInvites :: Bool
   , gatewayIntentVoiceStates :: Bool
-  , gatewayIntentPrecenses :: Bool
+  , gatewayIntentPresences :: Bool
```

Comment:
Haha I never noticed this, good spellcheck.

Might be a minor breaking change, but will generate a very nice compiler warning (`similar variable in scope: did you mean [...]`) so it'll be fine to release